Example/Marker/ThemeFactory.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ThemeFactory.swift
+//  Marker
+//
+//  Created by Htin Linn on 5/25/16.
+//  Copyright © 2016 Prolific Interactive. All rights reserved.
+//
+
+import UIKit
+
+internal struct ThemeFactory {
```

Comment:
since you're invoking the word "factory", i might rather expect factory functions. something like:

```swift
internal struct ThemeFactory {
    
    // MARK: - Instance functions
    
    func fontTheme() -> FontTheme {
        return MarkerExampleFontTheme(fonts: fonts())
    }
    
    // MARK: Private instance functions
    
    private func fonts() -> Fonts {
        return HelveticaFonts()
    }
    
}
```